### PR TITLE
docs: grammatical updates

### PR DESCRIPTION
**Pull request by QuinnyPig**
A bunch of non-substantive updates to the documentation to improve readability and flow.

### CODE REVIEW

File: demo/starter/slides.md [hunk 1]
```diff
@@ -57,13 +57,12 @@ transition: fade-out
 Slidev is a slides maker and presenter designed for developers, consist of the following features
 
 - 📝 **Text-based** - focus on the content with Markdown, and then style them later
-- 🎨 **Themable** - theme can be shared and used with npm packages
+- 🎨 **Themable** - themes can be shared and re-used as npm packages
 - 🧑‍💻 **Developer Friendly** - code highlighting, live coding with autocompletion
-- 🤹 **Interactive** - embedding Vue components to enhance your expressions
+- 🤹 **Interactive** - embedd Vue components to enhance your expressions
```
Comment (jakobkordez) [inline demo/starter/slides.md:62]: Typo, fix to embed

File: docs/guide/editors.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Editor Support
 
-Since Slidev is using Markdown as the source entry, you can use ANY editors you love to write it.
+Since Slidev is using Markdown as the source entry, you can use any editor you'd lik' to write your slides.
```
Comment (jakobkordez) [inline docs/guide/editors.md:3]: Typo, like

File: docs/guide/hosting.md [hunk 3]
```diff
@@ -73,7 +73,7 @@ $ slidev build --watch
 
 ### Multiple entries
 
-You can also build multiple slides at once.
+You can build multiple slide deckss at once.
```
Comment (jakobkordez) [inline docs/guide/hosting.md:76]: Typo, decks

File: docs/guide/install.md [hunk 4]
```diff
@@ -99,19 +99,19 @@ You can visit your slides at `http://localhost:3030/`
 
 ### Build hostable SPA (Single Page Application)
 
-Run command `docker exec -i slidev npx slidev build` on the running container `slidev`. It will generate static HTML files under `dist` folder.
+Run `docker exec -i slidev npx slidev build` on the running container `slidev`. It will generate static HTML files under `dist` folder.
 
 #### Host on Github Pages
 
-You can host `dist` in a static website such as [Github Pages](https://tangramor.github.io/slidev_docker/) or Gitlab Pages.
+You can host `dist` as a static website via such services as [Github Pages](https://tangramor.github.io/slidev_docker/) or Gitlab Pages.
```
Comment (jakobkordez) [inline docs/guide/install.md:106]: ... via services such ...

File: docs/guide/syntax.md [hunk 5]
```diff
@@ -100,19 +100,19 @@ This is a page with the layout `center` and a background image.
 
 ## Code Blocks
 
-One big reason I am building Slidev is the need to make my code look just right in the slides. So just as you expected, you can use Markdown-flavored code block to highlight your code.
+One big reason that led to the creation of Slidev is the need to make code look just right in  slides. Accordingly, you can use Markdown-flavored code blocks to highlight your code.
```
Comment (jakobkordez) [inline docs/guide/syntax.md]: "just right in slides." is missing "the" => "just right in the slides."

File: docs/guide/syntax.md [hunk 6]
```diff
@@ -154,12 +154,12 @@ function add(
...
 
 ::: tip
-Learn more in the [clicks animations guide](./animations#positioning).
+Learn more in the [click animations guide](./animations#positioning).
 :::
 
 ### Line Numbers
 
-You can enable line number to all slides by setting `lineNumbers: true` on the config or enable each code block individually by setting `lines: true`. You can also set the starting line for each code block and highlight the lines accordingly, defaults to 1:
+You can enable line numbers for all slides by setting `lineNumbers: true` in the global config or enable each code block individually by setting `lines: true`. You can also set the starting line for each code block and highlight the lines accordingly; it defaults to 1:
```
Comment (jakobkordez) [inline docs/guide/syntax.md:162]: line numbering

File: docs/guide/syntax.md [hunk 7]
```diff
@@ -377,7 +377,7 @@ blockquote {
 
 Just like you would do in markdown, you can use images pointing to a remote or local URL.
 
-For remote assets, the built-in [`vite-plugin-remote-assets`](https://github.com/antfu/vite-plugin-remote-assets) will cache them into the disk at the first run so you can have instant loading even for large images later on.
+For remote assets, the built-in [`vite-plugin-remote-assets`](https://github.com/antfu/vite-plugin-remote-assets) will cache them onto disk at first run so you can have instant loading--even for large images later on.
```
Comment (jakobkordez) [inline docs/guide/syntax.md:380]: will cache them onto **the** disk at first

**Review by antfu — APPROVED** [review]
Thanks!

**Comment by jakobkordez** [comment]
Bad merge, full of typos

**Review by jakobkordez — CHANGES_REQUESTED** [review]

File: docs/guide/syntax.md (hunk 7 above)
Comment (QuinnyPig) [reply docs/guide/syntax.md:380]: "Cache to disk" would arguably be the most correct if we're playing inside baseball with storage admins. :D

**Pull request merged**